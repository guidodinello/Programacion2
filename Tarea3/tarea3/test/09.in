# Prueba esAvl, insertarEnBinario, removerDeBinario, imprimirBinario, cantidadBinario, alturaBinario
insertarEnBinario (10,10.10)
esAvl
insertarEnBinario (5,5.5)
esAvl
insertarEnBinario (20,20.20)
esAvl
esAvl
insertarEnBinario (2,2.2)
esAvl
insertarEnBinario (7,7.7)
esAvl
removerDeBinario 7 
insertarEnBinario (1,1.1)
esAvl
removerDeBinario 2 
removerDeBinario 1 
insertarEnBinario (7,7.7)
insertarEnBinario (9,9.9)
esAvl
removerDeBinario 9 
insertarEnBinario (6,6.6)
esAvl
insertarEnBinario (2,2.2)
esAvl
imprimirBinario
cantidadBinario
alturaBinario
removerDeBinario 20 
esAvl
removerDeBinario 2 
esAvl
removerDeBinario 6 
esAvl
removerDeBinario 7 
esAvl
removerDeBinario 5 
esAvl
removerDeBinario 10 
esAvl
imprimirBinario
cantidadBinario
alturaBinario
Fin
