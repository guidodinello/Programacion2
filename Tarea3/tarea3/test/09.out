1>#  Prueba esAvl, insertarEnBinario, removerDeBinario, imprimirBinario, cantidadBinario, alturaBinario.
2>Insertado.
3>AVL.
4>Insertado.
5>AVL.
6>Insertado.
7>AVL.
8>AVL.
9>Insertado.
10>AVL.
11>Insertado.
12>AVL.
13>Removido.
14>Insertado.
15>No AVL.
16>Removido.
17>Removido.
18>Insertado.
19>Insertado.
20>No AVL.
21>Removido.
22>Insertado.
23>No AVL.
24>Insertado.
25>No AVL.
26>
-(20,20.20)
(10,10.10)
--(7,7.70)
---(6,6.60)
-(5,5.50)
--(2,2.20)
27>6.
28>4.
29>Removido.
30>No AVL.
31>Removido.
32>No AVL.
33>Removido.
34>No AVL.
35>Removido.
36>AVL.
37>Removido.
38>AVL.
39>Removido.
40>AVL.
41>
42>0.
43>0.
44>Fin.
